react-frontend/src/components/WorkoutCard.js

```diff
@@ -6,6 +6,12 @@ import "./WorkoutCard.scss";
 
 // Create content in card
 function CardTable(props) {
+     console.log(props.item);
+     // Check if props.item is an array
```

Comment:
We resolved this issue right. I think there will be other failsafes in place after we move from local storage to databases.

Reply:
yeah but i don't think it hurts to have it right?